From: Robert Garrett <robertgarrett404@gmail.com>
Date: Wed, 9 Jul 2025 10:39:52 -0500
Subject: Don't specify user & group ownership in install

Non-root users can't assign files to be owned by the root user, so this
fails. Writing to '/usr/bin' is also restricted, so running as a non-root
user will *still* fail. Running it via `sudo` means the install already
runs in the context of the root user, so it's already the right permissions.

So don't specify the ownership at all.
---
 Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index 6152ebd..e2e2c61 100644
--- a/Makefile
+++ b/Makefile
@@ -18,7 +18,7 @@ clean:
 # `DESTDIR` is normally empty and will have no effect. It's for future Debian
 # packaging (dpkg-buildpackage installs to a fake root, and uses that to do it)
 install: $(EXE)
-	install -o root -g root -m 755 -C ./Series -D $(DESTDIR)/usr/bin/Series
+	install -m 755 -C ./Series -D $(DESTDIR)/usr/bin/Series
 
 # the executable depends on the sources existing
 $(EXE): $(SRCS)
